### PR TITLE
tc: add page

### PR DESCRIPTION
**Pull request by NimrodRak**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/tc.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`tc qdisc show dev eth0`
+
+- Delete all trafic control rules:
+
+`tc qdisc del dev eth0`
+
+- Change traffic control rule:
+
+`tc qdisc change dev eth0 root netem {{policy}} {{policy_parameters}}`
```
Comment (einverne) [inline pages/linux/tc.md:32]:
```suggestion
`tc qdisc change dev {{eth0}} root netem {{policy}} {{policy_parameters}}`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Hi @NimrodRak  Thank you for the new page. It seems that `tc` command is complex. I left some suggestions for you to take a look.

**Comment by NimrodRak** [comment]
yup missed those, thx!

**Review by einverne — APPROVED** [review]
Looks good now.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**